### ISSUE TITLE
process of adding a mods folder(pack.json optional)

### ISSUE THREAD

**Issue by EvilMelYTT** [ShadowMario/FNF-PsychEngine#10447]
### Describe your problem here.

I simply want to know the (in)direct process for adding a mods files pre or post html5 building/exporting

### Are you modding a build from source or with Lua?

Source

### What is your build target?

HTML5/Browser

**Comment by TheKitBoi** [comment]
First of all html5 dose not support html5
And second why would you want that to be an addition

**Comment by EvilMelYTT** [comment]
No, I'm asking for people to inform me on the process of adding mods into psych engine source code due to problems with exporting
